suite/tests/runall.cmake

```diff
@@ -103,7 +103,7 @@ function (do_sleep ms)
   else ()
     # XXX: ping's units are secs.  For now we always do 1 sec.
     # We could try to use cygwin bash or perl.
-    execute_process(COMMAND ${PING} 127.0.0.1 -n 1 OUTPUT_QUIET)
+    execute_process(COMMAND ${PING} 127.0.0.1 -n 2 OUTPUT_QUIET)
```

Comment:
It seems that the previous command only slept a few milliseconds.

Running the following batch, the first 5 pings only take a few milliseconds, while the last takes just over 5 seconds.
```
@echo %time%
ping 127.0.0.1 -n 1
ping 127.0.0.1 -n 1
ping 127.0.0.1 -n 1
ping 127.0.0.1 -n 1
ping 127.0.0.1 -n 1
@echo %time%
ping 127.0.0.1 -n 6
@echo %time%
pause
```

Reply:
Presumably the behavior changed across Windows versions.